libs/acn/LLRPProbeReplyPDUTest.cpp

```diff
@@ -176,6 +176,14 @@ void LLRPProbeReplyPDUTest::testPrepend() {
     0xff
   };
   OLA_ASSERT_DATA_EQUALS(expected_data, sizeof(expected_data), buffer, length);
+
+  // test null stack
+  LLRPProbeReplyPDU::PrependPDU(
```

Comment:
does this test actually test something? shouldn't it check the return value?

Reply:
There is no return value, it's a void. It only modifies stack, which we've passed in as null anyway, so no change there. I guess we could prove the stack was still null, but I'm not sure it's very beneficial.

Without the null fix in the code though, the test itself segfaults too, so we'll know its broken...